bin/hubot

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+PATH="node_modules/.bin:node_modules/hubot/node_modules/.bin:$PATH" \
```

Comment:
This is from the automatically generated boilerplate. Is it required? If not, let's remove it.